django/contrib/postgres/functions.py

```diff
@@ -3,17 +3,9 @@
 
 class RandomUUID(Func):
     template = 'GEN_RANDOM_UUID()'
-
-    def __init__(self, output_field=None, **extra):
```

Comment:
What about setting `arity=0` instead to prevent any expression from being passed to it?